ui/desktop/README.md

```diff
@@ -32,7 +32,7 @@ You will need the following things properly installed on your computer.
 * [Git](https://git-scm.com/)
 * [Node.js](https://nodejs.org/)
 * [Yarn](https://yarnpkg.com/)
-* [Ember CLI](https://ember-cli.com/)
+* [Ember CLI](https://cli.emberjs.com/release/)
```

Comment:
Change on README are done by the Ember upgrade script, IMHO the script shouldn't do that lol. Anyway, I learn during the process after runnig the Ember upgrade script, we should run our `yarn run doc:toc` to leave README as we usually have it 😉

Reply:
Yup makes sense, ran the script. I actually kept the new URL because it looks like that's the updated URL and the old one just redirects to it.